tests/em/tdem/test_TDEM_forward_Analytic.py

```diff
@@ -93,7 +93,7 @@ def halfSpaceProblemAnaDiff(
 class TDEM_SimpleSrcTests(unittest.TestCase):
     def test_source(self):
         waveform = EM.TDEM.Src.StepOffWaveform()
-        assert waveform.eval(0.) == 0.
+        assert waveform.eval(0.) == 1.
```

Comment:
huh, interesting. This is used to evaluate the initial conditions? Initially, my thought process here was to instead set `hasInitialFields` on the waveform to `True`, and use that as a trigger instead. Check-out the BaseWaveform and see what you think.